src/pages/MainPage.tsx

```diff
@@ -61,13 +63,15 @@ const MainPage = () => {
           </TabPanel>
         </Tabs>
 
-        <BottomCTA>
-          <Link to="/tteokguk/create" className={styles.link}>
-            <Button color="secondary.100" applyColorTo="background" className={styles.button}>
-              소원 떡국 만들기
-            </Button>
-          </Link>
-        </BottomCTA>
+        <Link to={isLoggedIn ? "/tteokguk/create" : "/login"}>
```

Comment:
```suggestion
        <Link to={isLoggedIn ? "/tteokguks" : "/login"}>
```
어? 이거 안바뀌어 있네요? "/tteokguk/creeate"가 아니라 "/tteokguks" 가 스펙 같습니다!

Reply:
엥?? 그러면 소원 떡국 만들기 버튼을 클릭하면 페이지 이동이 일어나지 않는데요????

Reply:
아 헷갈렸네요! 무시해주십시오ㅠ